fahrschein-e2e-test/src/test/resources/log4j2.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Configuration status="warn">
+
+    <Appenders>
+        <Console name="stdout" target="SYSTEM_OUT">
+            <PatternLayout
+                    pattern="%date {%level} [%thread] [%X{X-Flow-ID}] [%logger] %message%n"/>
```

Comment:
I think you can drop x-flow-id here

Reply:
It's just a copy, shall we drop it from all log4j2.xml?

Reply:
(they're all test resources)

Reply:
Yes, I would say so.